t04_refactored.py

```diff
@@ -169,8 +169,53 @@ def team_7_adv(username, delay):
 
 
 def team_8_adv(username, delay):
-    pass
-    # TODO Add your code here
+    """
+    https://docs.google.com/document/d/1tpBnsiVxtpcC6sO9debi_r0aZpjHUot1gZv7d1fg4jg/edit?usp=sharing
+    Cody Bandy
+    CJ Spencer
+    :param username: the user's name
+    :param delay: float representing pauses in the program
+    :return: None
+    """
+    is_dead = False
+    is_free = False
+    is_stranded = False
+
+    print("\n")
+    print("Up ahead is a lit chamber with 3 doors, one to the East, one to the North, and one to the West.")
+    cardinal = input("Which door will you choose?")
+
+    if cardinal == "East":
+        # GOOD !!!
+        print("Fortune favors you! The door you have chosen is filled to the brim with treasures unimaginable in this world! Also, you see an exit just beyond the treasure!")
+        is_free = True
+        sleep(delay)
+
+    elif cardinal == "North":
+        # Might be a good choice
+        print("Behind the door lies a small snake that upon seeing you starts to grow!")
+        print("As you run away, the snake chases you as it continues to grow, expanding to fill everything behind you as your path ahead leads to a dead end...")
+        sleep(delay)
+        luck = random.randint(0, 10)
+        if luck <= 5:
+            print("There is no escape...")
+            is_dead = True
```

Comment:
This is not how we kill our user now. Instead, `return False` to kill them. See `main` for context.